go.mod

```diff
@@ -1,6 +1,5 @@
 module github.com/openGemini/openGemini
 
-go 1.20
```

Comment:
The semantics that specify the minimum version of Go required to compile the packages in this module are invalidated.

Reply:
Should we keep this? 
(https://github.com/openGemini/openGemini/issues/750#issuecomment-2362993092)

Reply:
Yes.